pytaxize/tax.py

```diff
@@ -181,7 +181,8 @@ def scrapenames(
     if len(method) > 1:
         sys.exit("Only one of url, file, or text can be used")
 
-    base = "http://gnrd.globalnames.org/name_finder.json"
+    # base = "http://gnrd.globalnames.org/name_finder.json"
+    base = "http://gnrd.globalnames.org/api/v1/find"
```

Comment:
```suggestion
    base = "https://finder.globalnames.org/api/v1/find"
```

I think this is the current url, yes?